config/routes.rb

```diff
@@ -11,6 +11,7 @@
     patch 'users/:user_id/add_patient/:id', to: 'users#add_patient', as: 'add_patient', defaults: { format: :js }
     patch 'users/:user_id/remove_patient/:id', to: 'users#remove_patient', as: 'remove_patient', defaults: { format: :js }
     post 'users/search', to: 'users#search', as: 'users_search', defaults: { format: :js }
+    patch 'users/clear_call_list/:id', to: 'users#clear_call_list', as: 'clear_call_list', defaults: { format: :js }
```

Comment:
A few changes to the routing here:

* can you have it patch to `users/clear_call_list`, without an id, since you're using `current_user` in the controller method?
* for maximum clarity on what's going on, can you have it route `as: 'clear_current_user_call_list'`?